lib/hawkular/alerts/alerts_api.rb

```diff
@@ -348,6 +348,8 @@ def initialize(trigger_hash)
       @context = trigger_hash['context']
       @type = trigger_hash['type']
       @tags = trigger_hash['tags']
+      @firing_match = trigger_hash['firingMatch']
+      @auto_resolve_match = trigger_hash['autoResolveMatch']
```

Comment:
Those params are parsed out of what the Hawkular server sends, which is 
camel case


Am 3. Juli 2016 3:42:28 nachm. schrieb abonas <notifications@github.com>:

>> @@ -348,6 +348,8 @@ def initialize(trigger_hash)
>>        @context = trigger_hash['context']
>>        @type = trigger_hash['type']
>>        @tags = trigger_hash['tags']
>> +      @firing_match = trigger_hash['firingMatch']
>> +      @auto_resolve_match = trigger_hash['autoResolveMatch']
>
> camel case is not very "Rubyish" (unlike Java), so unless these 2 
> parameters are coming from somewhere else that I missed, it is better to 
> have lowercase and "_" kind of format.
>
> ---
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/hawkular/hawkular-client-ruby/pull/97/files/1cfbcf0d658093c0bf03e3f8bafa9f01eda65736#r69391880